.github/workflows/ms.app.managedenvironments.yml

```diff
@@ -0,0 +1,147 @@
+name: 'Container Apps: Managed Environments'
```

Comment:
We recently updated the module workflow template for GitHub to leverage reusable workflows. Please use the most recent template and sorry for the inconvenience, you can reference for example [this one](https://github.com/Azure/ResourceModules/blob/main/.github/workflows/ms.analysisservices.servers.yml)